dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -108,7 +108,7 @@
                 <div class="col-sm-9">
                     <p:outputPanel id="filesButtons">
                     <div jsf:id="uploadComputeBlock" class="button-block pull-right no-margin-top"
-                          jsf:rendered="#{datasetFilesTab.canUpdateDataset() and !widgetWrapper.widgetView}">
+                          jsf:rendered="#{!widgetWrapper.widgetView}">
 
                         <!-- DOWNLOAD DCM SCRIPT BUTTON -->
                         <h:commandLink id="rsyncDLFF" actionListener="#{datasetFilesTab.downloadRsyncScript()}" styleClass="btn btn-default"
```

Comment:
For downloading rsync script we most likely need to have update dataset permissions (rsync script is used to upload file to repository)